src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -341,14 +353,17 @@ private void processExternSymbols() {
 
     // Enum values like Enum.A will appear as stand-alone symbols, but we do not need to emit them.
     externSymbolNames.removeAll(enumElementSymbols);
-    for (TypedVar symbol : externSymbols) {
-      if (enumElementSymbols.contains(symbol.getName())) {
-        externSymbols.remove(symbol);
+    Iterator<TypedVar> it = externSymbols.iterator();
+    while (it.hasNext()) {
+      if (enumElementSymbols.contains(it.next().getName())) {
+        it.remove();
       }
     }
 
     Set<String> shadowedSymbols = getShadowedProvides(externSymbolNames);
 
+    sortSymbols(externSymbols);
```

Comment:
Our tests run on JDK7, 8 and it seems that closure returns compiler.topSymbols() in different order between the two versions. We have only one golden file so the option is introduce two goldens or sort here.